bindings/src/account_history/msg/execute.rs

```diff
@@ -12,10 +12,7 @@ pub enum ExecuteMsg {
         delete_epoch: Option<u64>,
     },
     UpdateAccount {},
-    CleanHistory {
-        limit: u64,
-    },
-    CleanOldHistory {
+    CleanStorage {
```

Comment:
@cryptokage1996 as agreed in peer review, lets add a query to know the remaining length of items that needs to cleaned